ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/services/transaction/ApiTransactionService.scala

```diff
@@ -86,7 +86,7 @@ class ApiTransactionService private (
       .split(request.eventId.unwrap)
       .fold(
         Future.failed[TransactionTree](
-          Status.INVALID_ARGUMENT
+          Status.NOT_FOUND
```

Comment:
This should remain as `INVALID_ARGUMENT` because the eventID didn't match the expected format.